src/Client/Updater.php

```diff
@@ -52,23 +55,10 @@ public function __construct($repositoryName, $mirrors)
    * @see https://github.com/php-tuf/php-tuf/issues/21
    */
     public function refresh()
-    {
-    }
-
-  /**
-   * Validates a target.
-   *
-   * @param $targetRepoPath
-   * @param $targetStream
-   *
-   * @return bool
-   *   Returns true if the target validates.
```

Comment:
Hm, why are we deleting this?

Or is this just a diff artifact?

Reply:
All the things we've done so far do not happen when the application validates a single target, but rather when the application checks the repository for updates. The Python library has a `refresh()` function that applications call to begin an update check. So this is just moving the functionality to the right place, inside our `refresh()`

Now `refresh()` isnt empty and `validateTarget` is. The diff is weird.